exercises/luhn/canonical-data.json

```diff
@@ -51,17 +56,12 @@
       "expected": false
     },
     {
-      "description": "lots of zeros are valid",
-      "input": " 00000",
-      "expected": true
-    },
-    {
-      "description": "another valid sin",
-      "input": "055 444 285",
+      "description": "more than a single zero is valid",
```

Comment:
Updated to state the limitation here more clearly, and moved the space character to indicate that a "chain" of zeros is not required.